src/plans/create/steps/general-information/GeneralInformationStep.tsx

```diff
@@ -0,0 +1,106 @@
+import React, { FC, useState } from 'react';
+import { useFormContext, useWatch } from 'react-hook-form';
+import ProviderSelect from 'src/plans/components/ProviderSelect';
+
+import ControlledFormGroup from '@components/common/ControlledFormGroup';
+import { WizardStepContainer } from '@components/common/WizardStepContainer';
+import { V1beta1Provider } from '@kubev2v/types';
+import { Form, FormSection, MenuToggleStatus, TextInput } from '@patternfly/react-core';
+import { getInputValidated } from '@utils/form';
+import { useForkliftTranslation } from '@utils/i18n';
+
+import { GeneralFormFieldId, generalFormFieldLabels } from './constants';
+import { PlanProjectField } from './PlanProjectField';
+import { TargetProjectField } from './TargetProjectField';
+
+export const GeneralInformationStep: FC = () => {
+  const { t } = useForkliftTranslation();
+  const {
+    formState: { errors },
+    setValue,
+  } = useFormContext();
+  const targetProject: string = useWatch({ name: GeneralFormFieldId.TargetProject });
+  const [providerNamespace, setProviderNamespace] = useState<string>();
+
+  return (
+    <WizardStepContainer title={t('General')}>
+      <Form>
+        <FormSection title={t('Plan information')} titleElement="h3">
+          <p>{t('Name your plan and choose the project you would like it to be created in.')}</p>
+
+          <ControlledFormGroup
```

Comment:
you already extracted 2 other ControlledFormGroup , u don't want to keep this pattern and extract other ControlledFormGroup to a new component?